src/modules/About/About.tsx

```diff
@@ -28,6 +28,9 @@ const Container = styled(SectionContainer)`
   flex-direction: column;
   padding-bottom: 72px;
   padding-top: 49px;
+  @media (max-width: 768px) {
+    padding: 0px 40px 20px;
+  }
 `;
 
 const TopContainer = styled.div`
```

Comment:
I would raname TopContainer to a more declarative name, that explains what it does, it's a horizontal container, or row container
the name TopContainer doesn't provide the reader with info about what it does